extensions/amp-consent/0.1/test/test-consent-state-manager.js

```diff
@@ -251,26 +252,30 @@ describes.realWin('ConsentStateManager', {amp: 1}, env => {
               'amp-consent:test', composeStoreValue(consentInfo));
         });
 
-        it('new consent string always override old one', function* () {
+        it('do not store consentstring with dismiss consent state',
+            function* () {
+              instance.update(CONSENT_ITEM_STATE.DISMISSED, 'old');
+              yield macroTask();
+              expect(storageSetSpy).to.not.be.called;
+            });
+
+        it('new consent string override old one', function* () {
```

Comment:
Can we add another test, that we shouldn't attempt to store a string if we got the same response? 😄

Reply:
Does this work? https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/0.1/test/test-consent-state-manager.js#L277

Reply:
Yes that does! 😄 👍